Automattic/jetpack
# Super Cache: Boost banner with auto-install

Update: A previous iteration of this PR included an attempt to merge https://github.com/Automattic/jetpack/pull/26890 to add a new top level menu structure and a one-click auto-install for Jetpack Boost, but that turned into a quagmire with the main codebase not yet passing linting. So, I am breaking that up into multiple projects.

This PR brings a new dismissable one-click installation banner inviting users to try Jetpack Boost for further speed-ups:

![image](https://user-images.githubusercontent.com/1369626/205599091-b305073a-c2b7-4935-aaca-bfc885f77b0f.png)

Just click the button and it will auto-install and activate Jetpack Boost. Or, click the X to get rid of the notice.

#### Changes proposed in this Pull Request:
* Add a dismissable banner for boost to super cache.
* Automatically install Jetpack Boost with one click.

#### Other information:

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Make sure you can dismiss the banner.
* Make sure that clicking the link auto-installs Boost properly.
* Make sure the banner does not show if you already have Boost.
* If you have Boost installed but not activated, make sure the banner instead shows an option to activate Boost.
